aggregation/quantile/cm/stream.go

```diff
@@ -311,7 +345,9 @@ func (s *stream) addToMinHeap(heap *minHeap, value float64) {
 	// and return the current heap to the pool
 	if len(curr) == cap(curr) {
 		newHeap := s.floatsPool.Get(2 * len(curr))
-		newHeap = append(newHeap, curr...)
+		for i := 0; i < len(curr); i++ {
```

Comment:
Yeah it doesn't allocate a new slice (see https://play.golang.org/p/MUxy9zCY9W), which is somewhat surprising in itself.